### ISSUE TITLE
Better documentation for Elements

### ISSUE THREAD

**Issue by jbaechtelMT** [Badgerati/Pode#988]
Hard to follow documentation when Elements don't have clear; concise parameter explanations and how they are used.  Examples (for each parameter) would be great.

This is an example of what we see when looking up an Element and its' parameters (missing description):

-ListenerType

**_{{ Fill ListenerType Description }}_**

Type: String
Parameter Sets: (All)
Aliases:

Required: False
Position: Named
Default value: [string]::Empty
Accept pipeline input: False
Accept wildcard characters: False

**Comment by Badgerati** [comment]
Hi @jbaechtelKoala9126,

`-ListenerType` missing a description is actually a bug, so I'll fix that one. Do you have any more examples like this one? Because they should all have descriptions in the function/parameter summaries 🤔 

For documenting examples for every parameter is a massive task. I update and add to the docs every now and then, mostly for the most common use cases, and especially when people point out specific sections that could do with better descriptions/examples.

Mostly when it comes to updating the documentation the best approach is either:
* Raise a `documentation` issue, like this one, that points out a specific section you feel needs more clarification, better example, missing parameter docs, etc.
* Fork Pode and update the section, then raise a Pull Request to `develop` for review/approval
